src/xtgeo/xyz/_xyz.py

```diff
@@ -127,8 +127,7 @@ def describe(self, flush=True):
     # Import and export
     # ==================================================================================
 
-    @abc.abstractmethod
-    def from_file(self, pfile, fformat="guess"):
+    def from_file(self, pfile, fformat="xyz"):
```

Comment:
The `fformat` was the only difference between the base method and the re-implementation in the children. So technically this is a change in behavior, but _should_ not actually result in different behavior unless someone outside `xtgeo` is using `XYZ`.